desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,24 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Upgrades
+
+- [Buildx 0.8.1](https://github.com/docker/buildx/releases/tag/v0.8.1)
+
+### Bug fixes and minor changes
+
+- Fixed diagnostics upload when there is no HTTP proxy set.
+- Remove a false positive "vm is not running" error from self-diagnose.
```

Comment:
Maybe
```suggestion
- Removed a false positive "vm is not running" error from self-diagnose.
```
?

Also there is https://github.com/docker/for-mac/issues/6233 to link to.